cmake/Modules/LocalAom.cmake

```diff
@@ -38,6 +38,11 @@ if(EXISTS "${LIB_FILENAME}")
     set(_AOM_PC_LIBRARIES "${_AOM_STATIC_LIBRARIES}")
     # remove "aom" so we only have library dependencies
     list(REMOVE_ITEM _AOM_PC_LIBRARIES "aom")
+    # Work around crbug.com/aomedia/356153293: aom.pc has -lm in the
+    # Libs.private field on all platforms, whether libm or m.lib exists or not.
+    if(WIN32 OR APPLE)
+        list(REMOVE_ITEM _AOM_PC_LIBRARIES "m")
+    endif()
```

Comment:
The condition I used here is the inverse of the condition used in libaom's CMakeLists.txt:

```
if(NOT WIN32 AND NOT APPLE)
  target_link_libraries(aom ${AOM_LIB_LINK_TYPE} m)
  if(BUILD_SHARED_LIBS)
    target_link_libraries(aom_static ${AOM_LIB_LINK_TYPE} m)
  endif()
endif()
```

So the aom libraries are linked with `-lm` if `NOT WIN32 AND NOT APPLE` is true.